From 30343500394f74608873cb4f346de55d55300f8a Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 8 Oct 2014 14:04:44 -0400
Subject: [redhat] kabi: Adding symbol dev_err ()

Message-id: <241900d24f5ab359d3dedfeaf49ab60d372201f2.1412771859.git.poros@redhat.com>
Patchwork-id: 97177
O-Subject: [RHEL7.1 PATCH 24/67] kabi: Adding symbol dev_err ()
Bugzilla: 1148030
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1148030
    https://bugzilla.redhat.com/show_bug.cgi?id=1148030

    Description:
    ============
    These are just conventient forms of printk(), if alternatives
    are considered preferable that isn't a big deal.
    Symbol used in gru.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/dev_err
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/dev_err
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/dev_err

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/dev_err b/redhat/kabi/kabi-module/kabi_ppc64/dev_err
new file mode 100644
index 0000000..ee8f8d0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/dev_err
@@ -0,0 +1,2 @@
+#1-
+0x6724aeb0 dev_err vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/dev_err b/redhat/kabi/kabi-module/kabi_s390x/dev_err
new file mode 100644
index 0000000..07f420a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/dev_err
@@ -0,0 +1,2 @@
+#1-
+0xc1491476 dev_err vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/dev_err b/redhat/kabi/kabi-module/kabi_x86_64/dev_err
new file mode 100644
index 0000000..270f8f8
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/dev_err
@@ -0,0 +1,2 @@
+#1-
+0x0d795224 dev_err vmlinux EXPORT_SYMBOL
-- 
1.7.1